Documentation/Storage-Configuration/Ceph-CSI/ceph-csi-drivers.md

```diff
@@ -35,6 +35,87 @@ example, if the Rook operator is running in the namespace `my-namespace` the
 provisioner value should be `my-namespace.rbd.csi.ceph.com`. The same provisioner
 name must be set in both the storageclass and snapshotclass.
 
+search for `# csi-provisioner-name` in the [csi
+example](../deploy/examples/csi/) folder and update the name of the
+`provisioner`/`driver` as required.
+
+The following `sed` command can be used to update the provisioner name in the
```

Comment:
In the interest of simple docs, I'd suggest not even including this example. Maintaining this sed will be difficult too.